### PR TITLE
Andrei/new project setup

### PR DESCRIPTION
**Pull request by AndreiBozantan**

### CODE REVIEW

**Comment by dandanlen** [comment]
Hey @AndreiBozantan, I'm a bit confused by this project setup - we have all the project root files (.git, README, LICENSE) separate from the rest of the project configuration (.vscode, .devcontainer, .explorer.sln)... it would make more send to me to put everything at the same level. 

This also means that when I open the project in vs code, I can't easily edit the README.md for example. 

I had a look for .net project structure guidelines and found [this](https://gist.github.com/davidfowl/ed7564297c61fe9ab814)

Following the above would mean something more like: 

```
/
|- .git
|- .gitignore
|__.vscode/
|  |- [...]
|__.devcontainer/
|  |- [...]
|- explorer.sln
|- explorer.ruleset
|- LICENSE
|- README.md
|- [...]
|__/src
|  |__ /explorer.api
|  |  |- explorer.api.csproj
|  |  |- [...]
|  |__ /aircloak.client
|  |  |- airlcoak.client.csproj
|  |  |- [...]
|__/tests
|  |- [...]
```

This way all the project configuration would be accessible in the root folder of the project.

What do you think?

**Comment by dandanlen** [comment]
Other than that, I agree that using a vs code workspace was unnecessary, and using a .sln file seems to be a good idea too.

**Comment by AndreiBozantan** [comment]
I agree with this, so I will make the changes.

**Comment by AndreiBozantan** [comment]
@dandanlen ready for re-review

File: .dockerignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 **/.classpath
+**/.devcontainer
```
[the remark this answers is not in the document]
Comment (AndreiBozantan) [reply .dockerignore]: This is in the dockerigonre file in order to ignore it when files are copied in the Docker image file used for building and deploying the project. I think that the .devcontainer folder doesn't contain anything that will be useful during automated build or runtime.

[the remark this answers is not in the document]
Comment (dandanlen) [reply .dockerignore]: Actually, ignore this comment... I thought I was looking at .gitignore...

**Pull request merged**